package.json

```diff
@@ -70,7 +70,7 @@
   "dependencies": {
     "@microsoft/1ds-core-js": "^3.2.13",
     "@microsoft/1ds-post-js": "^3.2.13",
-    "@parcel/watcher": "2.1.0",
+    "@parcel/watcher": "2.5.0",
```

Comment:
Commenting at the wrong line due to github UI,

But there is a similar override in this file which can be removed.